cmake/EthCheckCXXCompilerFlag.cmake

```diff
@@ -10,9 +10,7 @@ include(CheckCXXCompilerFlag)
 #
 function(eth_add_cxx_compiler_flag_if_supported FLAG)
   # Remove leading - or / from the flag name.
-  string(REGEX REPLACE "^[-/]" "" name ${FLAG})
-  # Deletes any ':' because it's invalid variable names.
-  string(REGEX REPLACE ":" "" name ${name})
```

Comment:
We can't just remove this, this will break windows builds under some circumstances.